index.js

```diff
@@ -44,14 +44,14 @@ function addLayers(geojson, sourceId, layers) {
                 case 'LineString':
                     if (!geojson.properties) geojson.properties = {};
                     geojson.properties._id = cuid();
-                    layers.push(makeLayer(geojson, sourceId, 'LineString'));
+                    layers.unshift(makeLayer(geojson, sourceId, 'LineString'));
                     break;
                 case 'MultiPolygon':
                 case 'Polygon':
                     if (!geojson.properties) geojson.properties = {};
                     geojson.properties._id = cuid();
-                    layers.push(makeLayer(geojson, sourceId, 'LineString'));
-                    layers.push(makeLayer(geojson, sourceId, 'Polygon'));
+                    layers.unshift(makeLayer(geojson, sourceId, 'Polygon'));
+                    layers.unshift(makeLayer(geojson, sourceId, 'LineString'));
```

Comment:
I'm not sure if this will always guarantee a properly sorted feature collection.